rosario_angelica/countries.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Countries of the World</title>
+  <link rel="stylesheet" href="design.css">
+</head>
+<body>
+  <div class="section">
+    <h1>Countries of the World</h1>
+    <div class="search-container">
+      <input type="text" id="country-input" placeholder="Enter country name">
+      <button id="search-btn">Search</button>
+    </div>
+    <div id="country-details"></div>
+    <div id="region-countries"></div>
+  </div>
+  <script src="cjs.js"></script>
```

Comment:
Is this really your file name? Upon checking, your script is **country.js**  kindly check this so that your html will render the correct js file.